AncientReligions/history/characters/irish.txt

```diff
@@ -23509,7 +23509,10 @@
 	culture="irish"
 	father = 7258
 	550.1.1={
-		birth="550.1.1" 
+		birth="550.1.1"
+		effect = {
+			set_global_flag = ancrel
```

Comment:
Nice trick ! I missed the had_global_flag = { flag = ancrel days = 1 } # only on first day

Maybe the flag should be called "acr_startup_timer", or something like that to make it clearer ?